tests/settings.py

```diff
@@ -105,19 +105,19 @@
 OPENWISP_ADMIN_THEME_LINKS = [
     {
         'type': 'text/css',
-        'href': '/static/admin/css/openwisp.css',
+        'href': 'admin/css/openwisp.css',
```

Comment:
There's a good number of instances around which are deployed this way and it will be quite painful to update, so we must ensure backward compatibility.

Can you handle this in the preprocessing function? It should be as simple as using the string replace method, eg:

```python
static_file = static_file.replace('/static/, '')
```